manifests/compose.pp

```diff
@@ -56,6 +56,7 @@
 
   $docker_compose_location           = "${install_path}/${symlink_name}${file_extension}"
   $docker_compose_location_versioned = "${install_path}/docker-compose-${version}${file_extension}"
+  $docker_plugins_path               = '/usr/local/lib/docker/cli-plugins'
```

Comment:
AFAICT this path is different on RHEL/Rocky/CentOS:

```
$ rpm -ql docker-compose-plugin-2.16.0-1.el7.x86_64
/usr/libexec/docker/cli-plugins/docker-compose
...
```

So maybe this should be added to `params.pp` instead?